### PR TITLE
TOMEE-2984 Translate to Portuguese: examples/simple-singleton

### PR DESCRIPTION
**Pull request by Daniel-Dos**
- Translate to Portuguese
- fix in format code Java in Spanish

link to jira: https://issues.apache.org/jira/browse/TOMEE-2473

### CODE REVIEW

**Comment by jrxxjr** [comment]
The translation is ok

**Comment by Daniel-Dos** [comment]
thanks @jrxxjr  for review : )

File: examples/simple-singleton/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,365 @@
+:index-group: Session Beans
+:jbake-type: page
+:jbake-status: status=published
+= Simples Singleton
+
+Como o nome indica, um `javax.ejb.Singleton`, é um bean de sessão com a garantia de que há no máximo uma instância no aplicativo.
```
Comment (alexferreiradev) [inline examples/simple-singleton/README_pt.adoc:6]: I think that the translation of `applicación` can be `aplicação` in pt. What do you think ?

File: examples/simple-singleton/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+    @Lock(WRITE)
+    public <T> T removeComponent(final Class<T> type) {
+        return (T) components.remove(type);
+    }
+}
+....
+
+A menos que explicitamente especificado na classe de bean ou um método, o padrão `@Lock` é `@Lock(WRITE)`. O código acima usa a anotação `@Lock(READ)` na classe do bean para alterar o valor padrão para que o acesso multithread seja concedido por padrão. Portanto, só precisamos aplicar a anotação `@Lock(WRITE)` aos métodos que modificam o estado do bean.
+
+Essencialmente, `@Lock(READ)` permite acesso multithread à instância do bean Singleton, a menos que alguém invoque um método `@Lock(WRITE)`. Usar `@Lock(WRITE)` garantirá que a thread que invoca o bean terá acesso exclusivo à instância do Singleton bean durante sua invocação. Essa combinação permite que a instância do bean use tipos de dados que normalmente não são seguros para thread. Muito cuidado deve ser tomado.três coisas:
```
Comment (alexferreiradev) [inline examples/simple-singleton/README_pt.adoc:116]: At the and of paragraph, there is: `.três coisas`. Remove that

File: examples/simple-singleton/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+
+No exemplo, vemos `ComponentRegistryBean` usando um `java.util.HashMap` que está fora de sincronia. Para acertar, nós fazemos três coisas:
+
+[arabic]
+. Encapsulamento Não expomos a instância HashMap diretamente; incluindo seus iteradores, conjunto de chaves, conjunto de valores ou conjunto de entrada.
```
Comment (alexferreiradev) [inline examples/simple-singleton/README_pt.adoc:121]: Put a dot after `Encapsulamento`

**Review by alexferreiradev — COMMENTED** [review]
I think that the code is correct, but I have not time and enough acknowledgment to review these. I  hope that help.

File: examples/simple-singleton/README_pt.adoc (hunk 3 above)
Comment (jrxxjr) [reply examples/simple-singleton/README_pt.adoc:121]: I had not noticed this error. This error went unnoticed. Missing the point, after the word "Encapsulamento". Are two words starting with a capital letter, "Encapsulamento" and "Não".

**Pull request merged**